sdk/metricsadvisor/perf-tests/ai-metrics-advisor/sample.env

```diff
@@ -0,0 +1,11 @@
+# Retrieve these values from a service instance
+# in the Azure Portal and the Metrics Advisor web portal.
+METRICS_ADVISOR_ENDPOINT=https://<resource name>.cognitiveservices.azure.com/
```

Comment:
Do you plan to support sovereign cloud?

Do they use same url pattern?

Reply:
Are we doing that for all languages?

Reply:
Updated the sample.env to accomodate all urls